README.md

```diff
@@ -24,7 +24,7 @@ huskyCI is an open-source tool that performs security tests inside CI pipelines
 
 The main goal of this project is to help development teams improve the quality of their code by finding vulnerabilities as soon as possible.
 
-huskyCI can perform static security analysis in Python ([Bandit][Bandit] and [Safety][Safety]), Ruby ([Brakeman][Brakeman]), JavaScript ([Npm Audit][NpmAudit] and [Yarn Audit][YarnAudit]) and Golang ([Gosec][Gosec]). You should check our [wiki](https://github.com/globocom/huskyCI/wiki/How-does-huskyCI-work%3F) to better understand how this tool could help securing your organization projects!
+huskyCI can perform static security analysis in Python ([Bandit][Bandit] and [Safety][Safety]), Ruby ([Brakeman][Brakeman]), JavaScript ([Npm Audit][NpmAudit] and [Yarn Audit][YarnAudit]) and Golang ([Gosec][Gosec]).also it can perform leak scan on your git repo ([GitLeaks][Gitleaks]) You should check our [wiki](https://github.com/globocom/huskyCI/wiki/How-does-huskyCI-work%3F) to better understand how this tool could help securing your organization projects!
```

Comment:
What do you think of using `. It can also audit repositories for secrets like AWS Secret Keys, Private SSH Keys, and many others using ([GitLeaks][Gitleaks])` instead? 😃